src/main/scala/co/datamechanics/delight/DelightStreamingConnector.scala

```diff
@@ -308,6 +310,7 @@ class DelightStreamingConnector(sparkConf: SparkConf) extends Logging {
       }
       heartbeatThread.start()
       logInfo("Started DelightStreamingConnector heartbeat thread")
+      logInfo(s"Application will be available once your app is terminated on Delight at this url: $delightURL/apps/$dmAppId")
```

Comment:
logInfo(s"Application will be available on Delight a few minutes after it completes at this url: $delightURL/apps/$dmAppId")